agent/fishing_agent.py

```
@@ -355,7 +446,27 @@ def play_minigame(self) -> bool:
+                    return True  # 进度条消失,小游戏结束(可能已经钓到)
+                # 开局阶段:进度条可能尚未渲染,等待其出现(此前分析链路慢,
+                # 无意中起到了等待作用;提速后必须显式等待)
+                if not dumped_debug:
+                    try:
+                        from PIL import Image
+                        dbg = os.path.join(
+                            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+                            "debug", "fishing_minigame_debug.png")
+                        Image.fromarray(np.asarray(screenshot)[..., :3][..., ::-1]).save(dbg)
+                        print(f"    🐞 已保存小游戏调试帧: {dbg}")
+                    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)